stories/Page/Page.Card.stories.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react'
+import * as React from 'react'
```

Comment:
Not a huge deal.. but we probably should do this for `.js` files. It throws errors in console (cause it tries to import `PropTypes` incorrectly or something).

This is totally the thing to do for `.tsx` files though. Confusing I know :(

Reply:
Ahh yeah, it was throwing errors on tsx files so I changed that one too, changed it back!